lib/bolt/config.rb

```diff
@@ -239,6 +263,18 @@ def update_from_file(data)
     end
     private :update_from_file
 
+    # Shallow merge config for individual plugins
+    def update_plugins(data)
```

Comment:
I believe this is equivalent to:

```rb
@plugins.merge!(data) do |key,val1,val2|
  v1.merge(v2)
end
```

The block will only be invoked for keys that exist in both hashes and is used to determine the merge strategy.

Reply:
Oh, I didn't realize merge accepted a block with both values. Ruby you fancy.